### ISSUE TITLE
Generation of TSV and NT should happen at the same step

### ISSUE THREAD

**Issue by deepakunni3** [Knowledge-Graph-Hub/kg-covid-19#178]
@justaddcoffee In the `merge.yaml`, we can define multiple outputs. 

Right now our Jenkins build creates TSV in one stage and converts to NT in another stage.
Was this on purpose? For the sake of pushing the TSV to S3 first?

The amount of time taken to load the entire TSV again into memory is significantly high.

**Comment by justaddcoffee** [comment]
thanks @deepakunni3! I didn't realize we could define multiple outputs. Let's definitely change merge.yaml to output both TSV and NT. Can you ping me when you have a moment to chat about this?

**Comment by deepakunni3** [comment]
for example in our `merge.yaml` we can define:

```yaml
destination:
   merged-kg-tsv:
      type: tsv
      filename: merged-kg
   merged-kg-nt:
      type: nt
      filename: merged-kg.nt
```

**Comment by justaddcoffee** [comment]
closed by #179
